src/index.js

```diff
@@ -166,13 +80,13 @@ class UglifyJsPlugin {
       }
 
       compilation.plugin('optimize-chunk-assets', (chunks, callback) => {
+        const uglify = new Uglify(this.options.parallel);
```

Comment:
`if (!this.options.parallel || (os.cpus().length - 1) === 0)` we need to use `uglify.minify()` as we do atm, without any worker related logic attached please :)